### DISCUSSION TITLE
Adding a search field in the filter dropdown

### DISCUSSION THREAD

**Discussion by mallanza** [rappasoft/laravel-livewire-tables#2020] [Q&A]
Hi, 
1. I am wondering if there is a way to add a search field to narrow down the dropdown results.
_Similar to Bootstrap Select:_
https://user-images.githubusercontent.com/2874325/38997831-97e12bbe-43ab-11e8-85f5-b8c05d91c7b1.gif

2. How can I add custom styles to filters? right now the dropdown is a bit narrow

```
       public function filters(): array
    {
        return [
            MultiSelectDropdownFilter::make('Driver')
                ->options(
                    Driver::query()
                        ->where('drivers.instance_id', $this->user->instance_id)
                        ->whereHas('truck') // Ensure the driver is linked to a truck
                        ->orderBy('first_name')
                        ->get()
                        ->keyBy('id')
                        ->map(fn($driver) => $driver->first_name)
                        ->toArray()
                )
                ->setFirstOption('All Drivers')
                ->filter(function (Builder $builder, array $values) {
                    $builder->whereHas('driver', fn($query) => $query->whereIn('drivers.id', $values));
                })
        ];
    }
```

**Comment by lrljoe** [comment]
This'd be via a Livewire Component Filter typically for complex use cases, there's a few updates coming in the next updates that will enhance this, after which some smoother examples will be provided.

Alternatively, you can use a custom view for a filter!
